src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -195,7 +195,7 @@ def main(
         logger.info(f"Found {len(job_parameters)} jobs to run.")
 
     num_workers = len(job_parameters)
-    if extra_args["max_workers"]:
+    if "max_workers" in extra_args:
```

Comment:
It's minor, but important fix! If "max_workers" isn't in extra_args, it's a key error!